app/2.1.x/hybrid-mode.md

```diff
@@ -97,11 +97,155 @@ In PKI mode, the Control Plane and Data Plane don't need to use the same `cluste
 Instead, Kong validates both sides by checking if they are from the same CA. This eliminates the risk of
 transporting private keys around.
 
+Typically, a CA certificate will look like this:
```

Comment:
Can we tab between these examples? They currently take up the majority of the page. Here's the outline:

{% navtabs %}
{% navtab CA Certificate Example %}
Typically, a CA certificate will look like this: 

[example]

{% endnavtab %}
{% navtab CA Certificate on CP %}
Here's an example of a typical CP certificate:

[example]

{% endnavtab %}
{% navtab CA Certificate on DP %}
Here's an example of a typical DP certificate:

[example]

{% endnavtab %}
{% endnavtabs %}